### PR TITLE
Refactoring Python code for better readability.

### PR DESCRIPTION
**Pull request by radojicic23**
Noting much, just refactoring all Python scripts for better readability.

### CODE REVIEW

**Comment by thibaultmthh** [comment]
add .idea to the gitignore no ?

**Comment by bweston92** [comment]
@radojicic23 Twitter has their own code style guidelines see https://github.com/twitter-archive/commons/blob/master/src/python/twitter/common/styleguide.md
@thibaultmthh to their own global gitignore yes.

File: .idea/.gitignore
Comment (rps2579) [inline .idea/.gitignore:1]: Aren't the files within .idea supposed to be ignored?

File: src/python/twitter/deepbird/projects/timelines/configs/recap_earlybird/feature_config.py
Comment (rps2579) [inline src/python/twitter/deepbird/projects/timelines/configs/recap_earlybird/feature_config.py:1]: The only change I notice here is the formatting change from default 2 spaces to 4 spaces. 

Is this necessary?

File: src/python/twitter/deepbird/projects/timelines/scripts/models/earlybird/train.py
Comment (rps2579) [inline src/python/twitter/deepbird/projects/timelines/scripts/models/earlybird/train.py:1]: Mostly looks to be change in indentation spacing from 2 to 4.

**Review by rps2579 — COMMENTED** [review]
I have noticed 3 types of changes in this PR. 
1. Indentation space changed from 2 to 4.
2. In some files, imports have been reordered
3. .idea files 

I have 2 questions. 
1. Aren't .idea files supposed to be ignored?
2. Do we need this indentation change?

P.S: Is this PR merging necessary?

**Review by devvsakib — COMMENTED** [review]
Awesome Work

**Pull request closed without merging**